
Work around the bad quality of compile testing of the project.

[   44s] hnb-test.c: In function 'hnb_test_nas_rx_auth_req':
[   44s] hnb-test.c:329:3: error: 'return' with no value, in function returning non-void [-Werror=return-type]
[   44s]   329 |   return;
[   42s] test_common.c:91:1: error: control reaches end of non-void function [-Werror=return-type]
[   42s]    91 | }

---
 src/tests/hnb-test.c    |    2 +-
 src/tests/test_common.c |    1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

Index: osmo-iuh-0.5.0/src/tests/hnb-test.c
===================================================================
--- osmo-iuh-0.5.0.orig/src/tests/hnb-test.c
+++ osmo-iuh-0.5.0/src/tests/hnb-test.c
@@ -326,7 +326,7 @@ static int hnb_test_nas_rx_auth_req(stru
 
 	if (len < sizeof(*ar)) {
 		printf("GSM48 Auth Req does not fit.\n");
-		return;
+		return -1;
 	}
 
 	printf(" :) Authentication Request :)\n");
Index: osmo-iuh-0.5.0/src/tests/test_common.c
===================================================================
--- osmo-iuh-0.5.0.orig/src/tests/test_common.c
+++ osmo-iuh-0.5.0/src/tests/test_common.c
@@ -88,6 +88,7 @@ int test_common_init(void)
 
 	log_set_print_filename(osmo_stderr_target, 0);
 	log_set_use_color(osmo_stderr_target, 0);
+	return rc;
 }
 
 void test_common_cleanup(void)
